### DISCUSSION TITLE
.

### DISCUSSION THREAD

**Discussion by Partabsaifzakir** [aclap-dev/video-downloadhelper#802] [Help and Q&A]

**Comment by Mattmjs** [comment]
It's another "unable to open key file". There has  been a few of those brought to the forum. It doesn't look like there is a solution.

**Comment by Partabsaifzakir** [comment]


**Reply by GrampaWildWilly** [reply]
Unless I just didn't spend enough time hunting around the site, it appears there is no Firefox version of this software.  Is there a license or subscription for CocoCut?  If it's not too much prying, what does it cost?

**Reply by Mattmjs** [reply]
They used to have a Firefox extension some time ago, I believe Mozilla banned it and booted it out of add on site.
What does that say about the extension ? There are chrome & edge extension versions.
Interesting are the reviews, someone says they feel robbed on the chrome store. On the edge store someone complaining
about a corrupted file.
